src/sacn/receiver.c

```diff
@@ -18,25 +18,30 @@
  *****************************************************************************/
 
 /*********** CHRISTIAN's BIG OL' TODO LIST: *************************************
- - Add EtcPalMcastNetintId to EtcPal, and remove struct SacnMcastNetintId and RdmnetMcastNetintId
- - I've added the universe to all the callbacks.  Make sure the notification structs are initialized and work correctly for all notifications.
- - Add full support for the sources found notification. Packets aren't forwarded to the application until the source list is stable.
+ - Add full support for the sources found notification. Packets aren't forwarded to the application until the source
```

Comment:
This list should be removed from the code before merging it to develop. If any to-do items remain, they should be tracked in Jira.

Reply:
I still saw some TODO CHRISTIAN notes in the source as well (e.g. reset_networking).  Those should be cleaned up before going into develop (hopefully by implementing, but JIRA tickets if not).